pkg/ccl/streamingccl/streamingest/alter_replication_job.go

```diff
@@ -219,12 +239,13 @@ func alterReplicationJobHook(
 				alterTenantStmt,
 			)
 		}
-
-		if tenInfo.PhysicalReplicationConsumerJobID == 0 {
-			return errors.Newf("tenant %q (%d) does not have an active replication job",
-				tenInfo.Name, tenInfo.ID)
-		}
 		jobRegistry := p.ExecCfg().JobRegistry
+		if !alterTenantStmt.Options.IsDefault() {
+			return alterTenantSetReplication(ctx, p.InternalSQLTxn(), jobRegistry, options, tenInfo)
+		}
```

Comment:
Not really your problem but this and the last couple of changes have moved this function from one in which all of the possible commands were handled in the if/else block below into one where some are handled above with early returns and some are handled below with early returns.  

If we find ourselves modifying this again it might be nice to clean up.